docs/Development.md

```diff
@@ -404,8 +411,14 @@ Running an exporter on your local machine should follow this process:
 
 At this point, your exporter should be available at http://localhost:8080
 
-    curl http://localhost:8080
+    curl http://localhost:8080/metrics
+
+7. (Optional) Webhook exporter
+
+Webhook type exporter has an additional URL target http://localhost:8080/pelorus/webhook, that allows to receive POST data, for example using curl:
```

Comment:
to run this command, user does not have to be at `exporters/tests/data`? I think it is good to add this info

Reply:
fixed.